src/components/Pokemon.js

```diff
@@ -0,0 +1,42 @@
+import React, {useEffect, useState} from "react";
+import axios from "axios";
+import './pokemon.css'
+
+function Pokemon({pokemoninfo}) {
+    const [pokemon, setPokemon] = useState({})
```

Comment:
Als je een makkelijk onderscheid wil hebben tussen of het al wél of niet is ingeladen, zou ik hier voor default waarde van `undefined` gaan
```suggestion
    const [pokemon, setPokemon] = useState();
```